src/app.js

```diff
@@ -1,9 +1,23 @@
 // ./src/app.js
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
 
-import App from './components/App';
+import App from "./components/App";
+
+import { createStore } from "redux";
+import { Provider } from "react-redux";
+import reducer from "./users/duck/reducers";
+
+const store = createStore(
+    reducer,
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+);
+
+window.store = store;
```

Comment:
Raczej chcemy ograniczyć takie globalne rozwiąznaia.
Mają Prvidera, możemy to zrobić inaczej. Poszukać w materiałach o connect.